ddtrace/contrib/internal/langgraph/patch.py

```diff
@@ -98,6 +98,87 @@ async def traced_runnable_seq_ainvoke(langgraph, pin, func, instance, args, kwar
     return result
 
 
+@with_traced_module
+def traced_runnable_seq_astream(langgraph, pin, func, instance, args, kwargs):
+    """
+    Typically, RunnableSeq.ainvoke() is called when being run as a task for its parent Pregel (CompiledGraph).
+    However, when using Pregel.astream_events() (inherited from langchain's Runnable), it calls RunnableSeq.astream()
+    instead.
```

Comment:
```suggestion
```
I don't think this is a necessary comment, it's fairly straightforward that we need to trace different methods (non-stream, streamed) to capture all invocations